client/app/components/Carousel/PokemonCarouselComponent.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import type { Key } from 'react';
+import { Button, Carousel } from 'react-daisyui';
+import { PokemonCarouselProps } from './CarouselTypes';
+
+const PokemonCarouselComponent = ({
+  loading,
+  pokemonSprites,
+}: PokemonCarouselProps): React.JSX.Element => {
+  return (
+    <>
+      {!loading && pokemonSprites.length > 0 && (
+        <Carousel
+          className="rounded-box"
+          buttonStyle={(value: string) => {
+            return <Button color="primary">{value}</Button>;
+          }}
+          display="sequential"
+        >
+          {pokemonSprites.map((sprite: string, i: Key) => (
+            <Carousel.Item
```

Comment:
Why there is Key type instead of simple number

Reply:
Indeed, in this case, we can use simply number type.